src/studio/src/designer/frontend/app-development/features/administration/types.ts

```diff
@@ -0,0 +1,59 @@
+import { IRepository } from '../../types/global';
+
+export interface IFetchInitialCommitAction {
+  url: string;
+}
+
+export interface IFetchInitialCommitFulfilled {
+  result: any;
```

Comment:
Typing

Reply:
Copied from old reducers, and would take too much time to fix now. Created issue https://github.com/Altinn/altinn-studio/issues/5613 to address this problem.